mods/forestry-tools/clinometer.lua

```diff
@@ -101,86 +106,87 @@ local function update_hud(player, hud, hudname, text)
 end
 
 local function update_clinometer(player)
```

Comment:
Given the number of strings being modified with this function, I wonder if we should be be running `collectgarbage()` in here somewhere to release memory? Maybe @lukasdragon has some better ideas.